package.json

```diff
@@ -1,14 +1,13 @@
 {
```

Comment:
The lockfile (`package-lock.json`) was re-generated with an older version of the lockfile format. This is probably due to using an older version of NPM when running the upgrade process. @mauriciovigolo could you use Node 16 and a recent version of NPM to regenerate this file?

Reply:
Updated